### PR TITLE
Added code of conduct to project.

### PR DESCRIPTION
**Pull request by dlabaj**
Added code of conduct to the project.

### CODE REVIEW

**Review by jenny-s51 — APPROVED** [review]
this is awesome! 👍

**Review by mfrances17 — APPROVED** [review]
LGTM

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+[PSB]: https://github.com/syndesisio/syndesis/wiki/Project-Steering-Board
```
Comment (srambach) [inline CODE_OF_CONDUCT.md:73]: Do you still want this link?

**Comment by srambach** [comment]
👍 Thanks for adding!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (dlabaj) [reply CODE_OF_CONDUCT.md:73]: nope I need to remove that.  Thanks.

**Pull request merged**